sections/main-password-footer.liquid

```diff
@@ -75,13 +75,12 @@
     {%- endif -%}
   </ul>
   {% capture shopify %}
-    <a class="shopify-link" href="//www.shopify.com" rel="nofollow" target="_blank" title="Create your own online store with Shopify">
+    <div hidden id="warning-text">Opens in a new window.</div>
```

Comment:
The approach we had for this on the rest of the pages was to include this element in `theme.liquid` and reference it where needed. 
In this case for the password page we could either just add it here like you have right now or in `password.liquid` so that it can be reused by any section if needed. 

I think I'd go with adding it in `password.liquid` so that if we need to reuse it in the future then we have easy access to it 🤔 

Here is how it looks on `theme.liquid`:
```html
<ul hidden>
  <li id="a11y-refresh-page-message">{{ 'accessibility.refresh_page' | t }}</li>
  <li id="a11y-new-window-message">{{ 'accessibility.link_messages.new_window' | t }}</li>
</ul>
```

Reply:
Would that be moving lines 78-84  https://github.com/pizzaz93/dawn/blob/5a04955b8247fee454e1cdc6c463e4340ea3a214/sections/main-password-footer.liquid#L78 to https://github.com/pizzaz93/dawn/blob/5a04955b8247fee454e1cdc6c463e4340ea3a214/layout/password.liquid#L165 either above or below the footer?

Then referencing it in this section in between the { capture shopify} conditionals as password.something?

Reply:
No, it would changing line 78 from this file to: 
```
<ul hidden>
  <li id="a11y-new-window-message">{{ 'accessibility.link_messages.new_window' | t }}</li>
</ul>
```

And then moving it to `password.liquid` and add it right above the closing `</body>` tag 👍 

The rest can stay in this file.

Reply:
Added that line to `password.liquid`.